docs/DEVELOPING.md

```diff
@@ -187,6 +187,154 @@ $ make ftest NAME=s3
+
+
+## Azure Blob Storage Connector
+
+The [Elastic Azure Blob Storage connector](https://github.com/elastic/connectors-python/blob/main/connectors/sources/azure_blob_storage.py) is provided in the Elastic connectors python framework and can be used via [build a connector](https://www.elastic.co/guide/en/enterprise-search/current/build-connector.html).
+
+### Availability and prerequisites
+
+⚠️ _Currently, this connector is available in **beta** starting in 8.7_.
+Features in technical preview are subject to change and are not covered by the service level agreement (SLA) of features that have reached general availability (GA).
+
+Elastic versions 8.6.0+ are compatible with Elastic connector frameworks. Your deployment must include the Elasticsearch, Kibana, and Enterprise Search services.
+
+### Setup and basic usage
+
+Complete the following steps to deploy the connector:
+
+1. [Gather Azure Blob Storage details](#gather-azure-blob-storage-details)
+2. [Configure Azure Blob Storage connector](#configure-azure-blob-storage-connector)
+
+#### Gather Azure Blob Storage details
+
+Collect the information that is required to connect to your Azure Blob Storage:
+
+- The account name & account key of Azure Blob Storage account.
+- The blob endpoint of Azure Blob Service.
+
+#### Configure Azure Blob Storage connector
+
+The following configuration fields need to be provided for setting up the connector:
+
+##### `account_name`
+
+The account name of Azure Blob Storage account. Default value is `devstoreaccount1`
+
+##### `account_key`
+
+The account key of Azure Blob Storage account. Default value is `Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==`
+
+##### `blob_endpoint`
+
+The blob endpoint of Blob Service provided by Azure Blob Storage. Default value is `http://127.0.0.1:10000/devstoreaccount1`
+
+##### `retry_count`
+
+The number of retry attempts after failed call to Azure Blob Storage. Default value is `3`
+
+##### `concurrent_downloads`
+
+The number of concurrent downloads for fetching content of Azure Blob Storage. Default value is `100`
+
+##### `enable_content_extraction`
+
+Whether the connector should extract file content from Azure Blob Storage. Default value is `True` i.e. the connector will try to extract file contents.
+
+#### Content extraction
+
+The connector uses the Elastic ingest attachment processor plugin for extracting file contents. The ingest attachment processor extracts files by using the Apache text extraction library Tika. Supported file types eligible for extraction can be found [here](https://github.com/elastic/connectors-python/blob/main/connectors/utils.py#L29).
+
+### Connector Limitations
+
+- As of now Lease data & Blob tier won’t be updated in the document as the timestamp of the blob is not getting updated. Issue can be found [here](https://github.com/elastic/connectors-python/issues/289).
```

Comment:
```suggestion
- Currently, Lease data and Blob tier won’t be updated in the document, because the blob timestamp is not updated. Issue can be found [here](https://github.com/elastic/connectors-python/issues/289).
```